plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeParquetStatisticsUtils.java

```diff
@@ -222,6 +240,37 @@ private static Map<String, Object> jsonEncode(Map<String, Optional<Statistics<?>
                 .collect(toImmutableMap(Map.Entry::getKey, entry -> entry.getValue().get()));
     }
 
+    public static Map<String, Object> toNullCounts(Map<String, Type> columnTypeMapping, Map<String, Object> values)
+    {
+        verify(columnTypeMapping.keySet().containsAll(values.keySet()), "columnTypeMapping should contains all keys of values");
```

Comment:
include the key sets in the message

also, would be nice to add a comment _why_ this is expected. it's not obvious to me

Reply:
btw instead of this check here, i'd rather have a non-null check on `type` after `Type type = columnTypeMapping.get(value.getKey());` line